docs/api/boligsameie.md

```diff
@@ -19,22 +19,30 @@ For generell informasjon om tjenestene se egne sider om:
 
 * [Bruk av tjenestene](../om/bruk.md)
 * [Sikkerhetsmekansimer](../om/sikkerhet.md)
+* [Systemtilgang](../om/systemtilgang.md)
 * [Feilhåndtering](../om/feil.md)
 * [Versjonering](../om/versjoner.md)
 * [Teknisk spesifikasjon](../om/tekniskspesifikasjon.md)
 
 ## Scope
 
-Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:innrapportering/boligsameie`
+Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:innrapporteringboligsameie`
 
 ## Delegering
 
 Tilgang til dette API-et kan delegeres i Altinn, f.eks. dersom leverandør benyttes for den tekniske oppkoblingen.
 
+## Systemtilgang
+
+Bruk av API-et krever systemtilgang, som er ny funksjonalitet i Maskinporten levert av Digdir. 
+Informasjon vedr. dette finnes [her](../om/systemtilgang.md). 
+
 ## Teknisk spesifikasjon
 
 URL-er til API-et, beskrivelsen av parameterne, endepunkter og respons ligger i Open API spesifikasjonen på
-SwaggerHub
+[SwaggerHub](https://app.swaggerhub.com/apis/skatteetaten/boligsameie-api/0.0.1)
+
+Oppbygning av URL-er og åpninger i en evt. brannmur er beskrevet her [URL-er](../om/url.md)
```

Comment:
Er det samme gjeldende for våre APIer som deling ang brannmur og IP?

Reply:
Bør nok kanskje få verifisert det. Kan høre med Anders B. Tenkte det var greit å linke til den da det som står under "sikkerhet" blir direkte feil for oss.

Reply:
Våre api'er havner under samme IP-range ja, så da stemmer nok det